server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -275,18 +286,20 @@ public NewIndexType build() {
             Object subFieldMapping = entry.getValue();
             if (subFieldMapping instanceof Map) {
               entry.setValue(
-                addFieldToMapping(
-                  (Map<String, String>) subFieldMapping,
-                  "term_vector", "with_positions_offsets"));
+                  addFieldToMapping(
+                      (Map<String, String>) subFieldMapping,
+                      "term_vector", "with_positions_offsets"));
             }
           });
         }
 
         multiFields.put(fieldName, ImmutableMap.of(
-          "type", "string",
-          "index", "not_analyzed",
-          "term_vector", termVectorWithPositionOffsets ? "with_positions_offsets" : "no",
-          "norms", ImmutableMap.of("enabled", "false")));
+            "type", FIELD_TYPE_KEYWORD,
+            "index", INDEX_SEARCHABLE_FOR_KEYWORD,
+            "term_vector", termVectorWithPositionOffsets ? "with_positions_offsets" : "no",
+            "norms",
+            ImmutableMap.of("enabled", "false") // ES 5: replace with "false"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)